tests/test_truncatecomposite.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+
+import unittest
+
+import dimod.testing as dtest
+from dimod import BinaryQuadraticModel, HigherOrderComposite
+from dimod import TruncateComposite, ExactSolver
+
+
+class TestConstruction(unittest.TestCase):
+    def test_10(self):
+        sampler = TruncateComposite(ExactSolver(), 10)
+        dtest.assert_sampler_api(sampler)
+        dtest.assert_composite_api(sampler)
+
+        self.assertEqual(sampler.parameters, {})
+
+
+class TestSample(unittest.TestCase):
+    def test_sampleset_shorter(self):
+        h = {'a': -4.0, 'b': -4.0, 'c': 0}
+        J = {('a', 'b'): 3.2}
+
+        sampler = TruncateComposite(ExactSolver(), 10)
+        samples = sampler.sample_ising(h, J)
+
+        # we should see 2**n rows
```

Comment:
It's not super obvious to me what `n` is referring to. Based on context, I believe it's `2**len(h)`, but I'm not sure because I know TruncateComposite has a parameter `n` as well.

Reply:
that's right, but those comments were erroneous and confusing.